samples/snippets/csharp/VS_Snippets_Misc/tpl_parallel/cs/simpleforeach.cs

```diff
@@ -1,40 +1,112 @@
 ﻿// <snippet03>
 using System;
-using System.IO;
-using System.Threading;
+using System.Collections.Concurrent;
```

Comment:
Our build system now expects a *.csproj to verify that this code compiles. Could you please add a new project file here:

*samples/snippets/csharp/VS_Snippets_Misc/tpl_parallel/cs/simpleforeach.csproj*

```xml
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net5.0</TargetFramework>
  </PropertyGroup>

</Project>
```

Reply:
Added !!